### DISCUSSION TITLE
Starting multi with already selected icons

### DISCUSSION THREAD

**Discussion by gustavool1** [csandman/chakra-react-select#314] [Q&A] [answered]
Hi guys, hope you're doing fine. My question is if it's possible to start the select multi with already selected items. For example i have this selected that it's for you to add your user on to a group, but when editing the group i want to the users that are already on that group apparead as selected on the select. Is that possible? i was not able to find this information on the docs.
![image](https://github.com/csandman/chakra-react-select/assets/64279418/e360a4c5-52ea-4c67-80f3-0c42507d5d0d)

**Comment by csandman** [comment, marked as answer]
Sure, just pass the options you'd like to be selected by default in the `value` prop:

```tsx
const MySelect = () => {
  const [selectedPeople, setSelectedPeople] = useState([
    { label: "Jorge", value: "Jorge" },
    { label: "Matheuw", value: "Matheuw" },
  ]);

  return (
    <Select isMulti value={selectedPeople} onChange={setSelectedPeople} />
  );
}
```

**Reply by gustavool1** [reply]
Thank you so much!
